src/js/components/CreatePost.jsx

```diff
@@ -152,88 +154,58 @@ export default class NewsFeed extends React.Component {
-        </div>
-      );
-    }
-
     return (
-      <div>
-        <div className="user-post">
-          <form className="user-post-form">
-            <div className="input">
-              <div className="left-input inline">
-                <textarea
-                  ref={(input) => { this.textInput = input; }}
-                  className="text-input border"
-                  maxLength={textPostLength}
-                  rows="4"
-                  type="text"
-                  placeholder={placeholder}
-                  onChange={this.changeText}
-                  value={text}
-                />
-                <div className="soc-submit inline">
-                  { preferences.facebook && fbIntegration &&
-                    <div className="inline">
-                      <button
-                        type="button"
-                        className={`btn soc ${facebookStatus}`}
-                        onClick={this.fbToggle}
-                      >
-                        <TiSocialFacebook size={34} />
-                      </button>
-                    </div>
-                  }
-                  { preferences.twitter && twIntegration &&
-                    <div className="inline">
-                      <button
-                        type="button"
-                        className={`btn soc ${twitterStatus}`}
-                        onClick={this.twToggle}
-                      >
-                        <TiSocialTwitter size={34} />
-                      </button>
-                    </div>
-                  }
-                  <button
-                    type="button"
-                    className="submit-form btn btn-primary secondaryBackground"
-                    onClick={this.createPost}
-                  >
-                    Submit Post
-                  </button>
-                </div>
-              </div>
-              <div className="right-input">
-                <Dropzone className="dropzone border" onDrop={this.onPhotoDrop} multiple={false}>
-                  {dropzoneDisplay(this.deletePhoto)}
-                </Dropzone>
-              </div>
-            </div>
-          </form>
-        </div>
-      </div>
+	  <BetterCard className="user-post">
+		<CardTitle
+		  title="Post to News Feed" />
+		<CardText>
+          <textarea
+            ref={(input) => { this.textInput = input; }}
+            className="text-input"
+            maxLength={textPostLength}
+            rows="4"
+            type="text"
+            placeholder={placeholder}
+            onChange={this.changeText}
+            value={text}/>
+        </CardText>
+		{ preview &&
+		  <CardMedia
+			aspectRatio="wide"
+			image={preview}>
+            <MdHighlightRemove className="photo-delete" onClick={this.deletePhoto} size={24} />
+          </CardMedia>
+        }
+		<CardActions>
+		  <Dropzone className="dropzone border" onDrop={this.onPhotoDrop} multiple={false}>
```

Comment:
I'm going to assume everything still styles correctly?
Is the dropzone class still a thing? It looks like it is dropped unless there is magic scss going on or it exists elsewhere